server/lib/github.js

```diff
@@ -0,0 +1,26 @@
+var config = require('config');
+const request = require('request');
+const Promise = require('bluebird');
+
+module.exports = {
+
+  fetchUser(username) {
+    const options = {
+      url: `https://api.github.com/users/${username}?client_id=${config.github.clientID}&client_secret=${config.github.clientSecret}`,
+      headers: {
+        'User-Agent': 'OpenCollective'
+      },
+      json: true
+    };
+    return new Promise((resolve, reject) => {
```

Comment:
this can probably be simplified to something like `return Promise.promisify(request, {multiArgs: true})(options).then(args => args[1])`.

Guessing from bluebird doc: http://bluebirdjs.com/docs/api/promise.promisify.html